lib/registrant.rb

```diff
@@ -0,0 +1,25 @@
+class Registrant
+attr_reader :name,
+            :age, 
+            :permit,
+            :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {
+      :written=>false,
+      :license=>false,
+      :renewed=>false
+    }
+  end
+
+  def permit?
+    @permit == true 
```

Comment:
Similar to above, you can just have this line read `@permit` and it will evaluate to true or false, no need to include `== true`